.github/ISSUE_TEMPLATE/first_rc_checklist.md

```diff
@@ -0,0 +1,30 @@
+---
+name: First Release Candidate Checklist (maintainer only)
+about: Checklist template for the first release of every series
+title: llvmlite X.Y.Zrc1 Checklist (FIXME)
+labels: task
+
+---
+
+
+## llvmlite X.Y.Z
+
+* [ ] Merge to `main`.
+    - [ ] "remaining Pull-Requests from milestone".
+* [ ] Merge change log changes.
+    - [ ] "PR with changelog entries".
+* [ ] Create X.Y release branch.
+* [ ] Build conda packages on buildfarm (check "upload").
```

Comment:
```suggestion
* [ ] Annotated tag vX.Y.Zrc1 on release branch
* [ ] Build conda packages on buildfarm (check "upload").
```

Reply:
done in 15cb7619b6775cd259e23c611c63e9b947fff78b